models/responses.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+// Responses is a collection of responses to questions submitted by the client
+type Responses struct {
+	List              map[string]string `json:"responses"`
```

Comment:
this is kinda strange -- to have the JSON key be `responses`

Reply:
why? have any suggestions?